docs/docs/themes/using-a-gatsby-theme.md

```diff
@@ -2,71 +2,55 @@
 title: Using a Gatsby Theme
 ---
 
-The quickest way to get started using a Gatsby theme is to use a starter that's configured to use the theme.
+While you can [get started quickly with a Gatsby theme starter](/docs/themes/getting-started/), you can also install a Gatsby theme directly to an existing Gatsby site. Gatsby themes are plugins, so we [use them just like any other Gatsby plugin](/docs/using-a-plugin-in-your-site/).
 
-For example, `gatsby-starter-blog-theme` is a theme starter for the `gatsby-theme-blog` package.
+## Installing a Theme
 
-A **regular Gatsby starter** creates a new Gatsby site that is preconfigured for a particular use case. The resulting site effectively forks off the starter — after it’s created, the site maintains no connection to the starter.
+Like any Gatsby plugin, Gatsby themes are Node.js packages, so you can install them like other packages in node using npm.
 
-A **Gatsby theme starter** creates a new Gatsby site that installs and configures one or more Gatsby themes. When a starter installs a theme, it maintains the connection to that theme as a standalone npm package.
+For example, `gatsby-theme-blog` is the official Gatsby theme for creating a blog.
 
-Installing the Gatsby blog theme starter is the same process as a regular Gatsby starter:
+To install it, in the root of you site, run
 
-```shell
-gatsby new my-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
+```js
+npm install --save gatsby-theme-blog
 ```
 
-## What does a theme starter do?
+## Theme Options
 
-The starter for the official Gatsby blog theme does the following:
+Depending on the theme, there may be theme options that can be configured via `gatsby-config.js`.
 
-### 1. The starter installs the theme and configures it
+For example, `gatsby-theme-blog` can take in 4 potential options: `basePath`, `contentPath`, `assetPath`, and `mdx`.
 
-When you use a starter that's built with a theme, you will often see that you're initially presented with a lighter weight `gatsby-config.js`. Themes start doing their magic when installed via the `plugins` array:
-
-```javascript:title=gatsby-config.js
```

Comment:
We should keep the title here on the code block, as it helps with code reading comprehension.